be/src/io/shared_buffered_input_stream.cpp

```diff
@@ -36,6 +38,12 @@ void SharedBufferedInputStream::SharedBuffer::align(int64_t align_size, int64_t
     }
 }
 
+std::string SharedBufferedInputStream::SharedBuffer::debug() const {
```

Comment:
debug_string

Reply:
change next time, some people is waiting for this pr

Reply:
> change next time, some people is waiting for this pr

This is not a justifiable rationale
I'm afraid urgency alone does not justify compromising on quality assurance.
Besides, it's merely a name change, it should only require a small time investment.

Reply:
I will submit another pr to rename it.